datagateway_api/src/datagateway_api/database/models.py

```diff
@@ -1151,6 +1545,23 @@ class SHIFT(Base, EntityHelper, metaclass=EntityMeta):
     )
 
 
+class TECHNIQUE(Base, EntityHelper, metaclass=EntityMeta):
+    __tablename__ = "TECHNIQUE"
+    __singularfieldname__ = "technique"
+    __pluralfieldname__ = "techniques"
+    __table_args__ = (Index("UNQ_TECHNIQUE_0", "NAME"),)
+
+    id = Column("ID", BigInteger, primary_key=True)
+    createId = Column("CREATE_ID", String(255), nullable=False)
+    createTime = Column("CREATE_TIME", DateTime, nullable=False)
+    endDate = Column("ENDDATE", DateTime, nullable=False)
```

Comment:
`endDate` is not part of this entity. Please can it be removed?

Reply:
Yes, removed